src/HashBus.Application.Commands/AnalyzeTweet.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace HashBus.Application.Commands
+{
+    using System;
+
+    public class AnalyzeTweet
+    {
+        public Tweet Tweet { get; set; }
+
+        public Guid SessionId { get; set; }

```

Comment:
`SessionId` should not be present on this command. See how it's only ever being written to, but not read. It's a hangover from the old `TweetReceived` event, where it is needed for `Twitter.Monitor.CatchUp` to know whether to do a catch up.